### DISCUSSION TITLE
Air Force Cosmonauts

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#736] [Ideas]
So, in #669, I fixed the oversight that the game showed GROUP as blank in Basic Training and as GROUP X in Cosmonaut History.  In Astronaut History, since there was room, I changed it to show GROUP USAF, and Cosmonaut History as GROUP A.F.

I got to thinking though: why not use the abbreviation for the Soviet Air Force, as long as it doesn't exceed 5 characters?  So I looked it up, and it looks like there were several groups in what we would have grouped in "the Air Force".  Sort of the main general one was VVS, and I think two of them were from there.  The other, Dolgov, was from the Airborne Forces (VDV, basically paratroopers), but we could overlook that.

My question is, would that just be too obscure?  Most people probably wouldn't recognize that abbreviation, VVS.  Or would that be ok?

![USAF](https://user-images.githubusercontent.com/3118014/213609274-9669d164-3cbf-42bf-9669-6f88f0d2d698.png)
![A F](https://user-images.githubusercontent.com/3118014/213609282-d72ce1f7-0919-43f5-b252-52582380c70c.png)

**Comment by hweimer** [comment]
I think that part of the game is to educate people about the space race, so I'd say go for it. Maybe the abbreviation can be explained in the newscast that enables the nauts.

**Comment by peyre** [comment]
Thanks!  I'll implement it, and maybe add to the Soviet newscast.  Currently it says 

THE MINISTRY OF DEFENSE HAS TRANSFERRED THREE TOP FIGHTER PILOTS TO THE SPACE PROGRAM.

I could change it to say they were transferred FROM THE SOVIET AIR FORCES (V.V.S.) TO THE SPACE PROGRAM.  Would that sound right?
